src/R/Components/Impl/PackageManager/Implementation/View/PackageList.xaml.cs

```diff
@@ -88,5 +91,17 @@ public partial class PackageList : UserControl {
             var package = GetPackage(e);
             Model?.UnloadAsync(package).DoNotWait();
         }
+
+        protected override AutomationPeer OnCreateAutomationPeer() {
```

Comment:
But we already fire `PropertyChanged` when `IsInstalled` or 'IsLoaded` are changing. Does automation skip them for some reason?